### PR TITLE
Ui documentation

### PR DESCRIPTION
**Pull request by nickdotreid**
Created a stub for MOTECH UI documentation.

### CODE REVIEW

**Comment by pgesek** [comment]
Please remove unrelated changes and unresolved conflicts

**Comment by nickdotreid** [comment]
Cleaned up the code, can someone test this?

File: docs/source/development/ui_guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+More text will be added.
+
+Typography
+==========
+Since the MOTECH UI is built off the Bootstrap framework, it shares the same typographical features. The base MOTECH font in 'Helvetica Neue' 14px.
+
+Iconography
+-----------
+The MOTECH UI uses the Font Awesome icon libaray and implements icons using CSS psudo-elements so that the HTML templates in MOTECH are clean, semantic, and easy to update from a single location.
```
Comment (pgesek) [inline docs/source/development/ui_guidelines.rst]: libaray -> library

Comment (pgesek) [inline docs/source/development/ui_guidelines.rst]: psudo -> pseudo

File: docs/source/development/ui_guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+MOTECH Logo
+-----------
+The only custom design asset in the MOTECH platform is the MOTECH logo. A SVG version of the :download:`MOTECH logo is available here, <motech_logo.svg>` incase you need to do something with it.
```
Comment (pgesek) [inline docs/source/development/ui_guidelines.rst]: incase -> in case

File: docs/source/development/ui_guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+Examples of accessiable link text would be: 'Alert module stop' or 'Alert module add data'.
+
+An important note is while this text should be here for non-sighted users, this can be hidden for sighted users.
+
+Buttons
+-------
+Buttons should only be used to represent an action that will make a change rather than a link that will navigate a user to a different section of the website.
+
+When adding icons to buttons, don't add extra markup for the button icon, instead use a css class that adds a psudo-element.::
```
Comment (pgesek) [inline docs/source/development/ui_guidelines.rst]: psudo -> pseudo

**Pull request closed without merging**